app/assets/javascripts/discourse/app/components/post-text-selection.gjs

```diff
@@ -32,6 +33,13 @@ function getQuoteTitle(element) {
   return titleEl.textContent.trim().replace(/:$/, "");
 }
 
+const CSS_TO_DISABLE_FAST_EDIT = [
```

Comment:
This is the list of classes that will be checked against the selection and will "disable" fast edit if they match.